Taskfile.yml

```diff
@@ -247,6 +247,7 @@ tasks:
     desc: "Format and lint documentation"
     cmds:
       - docker-compose run --rm markdownlint --fix --dot --ignore CHANGELOG.md '**/*.md'
+      - docker-compose run --rm textlint --fix --rule one-sentence-per-line '*.md'
```

Comment:
Should we use `**/*.md` to also check files in the `.github` directory?

Reply:
Thank you, fixed